python/paddle/fluid/dygraph/layer_object_helper.py

```diff
@@ -200,10 +200,20 @@ def append_activation(self,
         act_type = act.pop('type')
 
         tmp = self.create_variable_for_type_inference(dtype=input_var.dtype)
+
+        if act_type == 'relu':
+            mask = self.create_variable_for_type_inference(
+                dtype=core.VarDesc.VarType.UINT8)
+            mask.stop_gradient = True
+            act_type = 'relu2'
```

Comment:
You created relu2 op and if act_type == 'relu', replace it with the new 'relu2'. I guess it is for backward compatibility. Do we have a future plan to remove "2" and replace 'relu' completely?

Reply:
Replace with an automatic pass.